news/news.rst

```diff
@@ -0,0 +1,25 @@
+**Added:**
+
+* Users can use the environment variable "TEST_XPDACQ_BASE_DIR" to set the base directory
+
+**Changed:**
+
+* Shutter configure for PDF beamline changes to "Close" and "Open"
+
+* Experiment changes to version 2 databroker
+
+**Deprecated:**
+
+* <news item>
```

Comment:
is there a reason it is like this and not like:
```
**Deprecated** None
```
won't doing it your way print ``<news item>`` when it is compiled?

Reply:
This is the feature of the latest rever. I remembered that they update their template of the template.rst.